library/MedEx/API.php

```diff
@@ -2998,15 +2998,15 @@ public function MedExBank($stage)
 
                                                 <!-- Modal content-->
                                                 <div class="modal-content">
-                                                <div class="modal-header"  style="background-color: #0d4867;color: #fff;font-weight: 700;">
-                                                    <button type="button" class="close" data-dismiss="modal" style="color:#fff;opacity:1;box-shadow:unset !important;">&times;</button>
+                                                <div class="modal-header text-white" style="background-color: #0d4867;font-weight: 700;">
```

Comment:
Just realized the `font-weight: 700` is basically the equivalent of the `font-weight-bold` class so you might as well replace that.